specification/schema/tile.schema.json

```diff
@@ -66,7 +66,32 @@
         },
         "content": {
             "description": "Metadata about the tile's content and a link to the content. When this is omitted the tile is just used for culling.",
-            "$ref": "content.schema.json"
+            "$ref": "content.schema.json",
+            "deprecated": true
+        },
+        "contents": {
+            "type": "array",
+            "description": "An array of contents.",
+            "items": {
+                "$ref": "content.schema.json"
+            },
+            "minItems": 1
+        },
```

Comment:
Unfortunately we can't change `content` to be an array because that would be a breaking change. And it would be awkward to allow `content` to be either an object or an array of objects.

Reply:
Should we enforce that it should be only either `content` or `contents`, i.e. put these in a `"oneOf"`?

Reply:
Yes! I pushed a fix for that in https://github.com/CesiumGS/3d-tiles/pull/634/commits/ee3514cf9de0469cd61d71bf6d72c0dcfc92a665. If my understanding of JSON schema is correct this snippet below should allow just `content`, just `contents`, or neither.

```json
"oneOf": [
  {
    "required": [
      "content"
    ]
  },
  {
    "required": [
      "contents"
    ]
  },
  {}
]
```